notebooks/266-table-question-answering/266-table-question-answering.ipynb

```diff
@@ -0,0 +1,413 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #25.        ov_model_xml_path.parent.mkdir(parents=True, exist_ok=True)
</pre><p>if I right remember this step is not required, save_model can create nested directories</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1408/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
You are right, removed